StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -127,5 +148,28 @@ private static void AppendFakeClintBoulderSpecialOrderKey()
             var specialOrderKeys = specialOrderKeysField.GetValue();
             specialOrderKeys.Add("Clint2Again");
         }
+
+        public static void ChangeScheduleForMovie()
+        {
+            if (!_archipelago.SlotData.Mods.HasMod(ModNames.SVE))
+            {
+                return;
+            }
+            if (_archipelago.GetReceivedItemCount("Progressive Movie Theater") >= 2)
+            {
+                string[] charactersToModify = {"Claire", "Martin"};
+                foreach(var name in charactersToModify)
```

Comment:
Why not loop over the entries of "characterToSchedule", since it is these characters anyway?